extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -80,13 +88,15 @@ export class AmpSidebar extends AMP.BaseElement {
       this.element.setAttribute('aria-hidden', 'true');
     }
 
+    // Prepare for animation (will not animate the initial state).
+    this.element.classList.add('-amp-sidebar-animate');
```

Comment:
Not a big deal and we can keep since it's easy to change later. But where this will break is when a user may want to customize the transition - e.g. change the curve.

Otherwise, I also don't think it really does anything here. If we want to do support initial open case - we should just do it explicitly and set `transition: none` on the element before opening it initially.

Again, not a huge deal for now.